lib/c-api/Cargo.toml

```diff
@@ -39,6 +39,9 @@ paste = "0.1"
 # Commented out for now until we can find a solution to the exported function problem
 # wasmer-wasm-c-api = { version = "1.0.0-alpha4", path = "crates/wasm-c-api" }
 
+[dev-dependencies]
+inline-c = { git = "https://github.com/Hywan/inline-c-rs", branch = "master" }
```

Comment:
Any plan on having this released?

Asking this because we may want to have guarantees on the API being stable over time.

Reply:
Sure! For the moment, `inline-c-rs` just assume `clang` is installed and uses only that. I'm planning to move to [`cc-rs`](https://github.com/alexcrichton/cc-rs/) if I can hack the lib to do what I want, i.e. to compile for all platforms. (ping @alexcrichton)

Reply:
Done. I'm releasing the first version of `inline-c-rs`.